docs/library/urandom.rst

```diff
@@ -0,0 +1,65 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator.
+
+Given below are the methods that this module provides.
+
```

Comment:
would be worth adding a sentence about `[`, `]`, `(`, `)` meaning closed and open intervals

Reply:
also add this line to link to CPython docs:
```
|see_cpython_module| :mod:`python:random`.
```